src/v2/Apps/Consign/Routes/SubmissionFlow/Utils/validation.ts

```diff
@@ -44,17 +44,11 @@ export const artworkDetailsValidationSchema = yup.object().shape({
   units: yup.string().required(),
 })
 
-/*
-this validation currently not in use until we find a workaround
-formik is checking whether errorMessage found for a photo under the hood
-and invalidates form submission if errorMessage exists
-
-The check is currently moved to UploadPhotos.tsx's Button directly
-*/
 export const uploadPhotosValidationSchema = yup.object().shape({
   photos: yup
     .array()
     .min(1)
+    .transform(fields => fields.filter(c => !c.errorMessage))
```

Comment:
Is this solve the problem you described above?

Reply:
yes that does it 🥳